home/tsconfig.json

```diff
@@ -4,8 +4,8 @@
     "resolveJsonModule": true,
     "baseUrl": ".",
     "noEmit": true,
-    "paths": {
-      "@expo/vector-icons": ["react-native-vector-icons"]
-    }
```

Comment:
Not sure it's safe to remove this

Reply:
Seems to be fine (I think `@expo/vector-icons` may not have had typescript typedefs at the time this was added in d89649cfd9f3e310c8cc73b38ad55ab249a5917e). `yarn tsc` works and the types are correctly found. TypeScript is only used for `yarn tsc`.